ReactCommon/fabric/components/text/paragraph/ParagraphStateData.h

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#pragma once
+
+#include <react/attributedstring/AttributedString.h>
+#include <react/textlayoutmanager/TextLayoutManager.h>
+
+namespace facebook {
+namespace react {
+
+class ParagraphStateData;
+
+using SharedParagraphStateData = std::shared_ptr<const ParagraphStateData>;
+
+/*
+ * LocalData for <Paragraph> component.
+ * Represents what to render and how to render.
+ */
+class ParagraphStateData {
+public:
+ /*
```

Comment:
Delete default copy constructor and copy assignment.
`ParagraphStateData(ParagraphStateData const &data) = delete;`